### DISCUSSION TITLE
Glasses produce disturbing reflections that are detected as points of light.

### DISCUSSION THREAD

**Discussion by Blubbfisch** [opentrack/opentrack#1653] [Q&A] [answered]
Hello,

As described, my glasses produce reflections during certain head movements, which are interpreted as points of light by the camera and massively disrupt head tracking. Even in a completely dark room, the light from the monitor seems to be sufficient to create reflections. 

Is there a way to filter out these reflections via the settings?

Thanks and regards,

Marek

![trackhat_lichtpunkte2](https://user-images.githubusercontent.com/99812317/236700807-2e971855-611b-4fde-8f2a-c4a306d8f7ed.png)

**Comment by Slion** [comment]
I vaguely recall implementing something to help with that in EasyTracker. EasyTracker only works with Kinect though.

**Reply by Blubbfisch** [reply]
Okay, I'm using a hattracker camera.

**Reply by sthalik** [reply, marked as answer]
Reduce exposure. If it's brighter than the points then I can't do anything.

**Reply by sthalik** [reply]
Otherwise you need to mask parts of the camera's aperture.
